lib/src/components/NxSubmitMask/__tests__/NxSubmitMask.test.tsx

```diff
@@ -42,17 +41,18 @@ describe('NxSubmitMask', function() {
 
   it('passes the successMessage prop as the child of NxLoadingSpinner when the success prop is true', function() {
```

Comment:
This test name is no longer accurate

Reply:
> This test name is no longer accurate

https://github.com/sonatype/sonatype-react-shared-components/pull/182/commits/1b1a50bff29509928cd95a565554b8b8af968045